.claude/commands/fix.md

```
@@ -0,0 +1,20 @@
+# Fix
+
+Auto-fix all formatting and linting issues across all languages.
+
+**TypeScript/JavaScript:**
+
+1. `pnpm format` — Apply Prettier formatting
+2. Run ESLint with fix for each app that has lint configured
+
+**Rust:**
+
+3. `cargo fmt --all` — Format all Rust code
```

markdownlint-cli2 (0.20.0)

[warning] Ordered list item prefix
Expected: 1; Actual: 3; Style: 1/1/1

(MD029, ol-prefix)